src/frame/sdk/GroupApi.ts

```diff
@@ -86,17 +87,56 @@ export function get(groupID: string) {
     return GroupApiEndpoints.callGroupGetApi(groupID).map(formatDetailedGroupResponse);
 }
 
+/**
+ * Create the error strings for missing users
+ */
+function createErrorForInvalidUserList(userKeysResponse: UserKeyListResponseType, memberList: string[]) {
+    const existingUserIDs = userKeysResponse.result.map(({id}) => id);
+    return memberList.filter((userID) => !existingUserIDs.includes(userID));
+}
+
+/**
+ * Checkes that all members listed in member list are existing members
+ */
+function validateMemberList(memberList: string[]) {
+    return memberList !== undefined && memberList.length > 0
+        ? UserApiEndpoints.callUserKeyListApi(memberList).flatMap((userKeysResponse) => {
+              if (userKeysResponse.result.length !== memberList.length) {
+                  const missingUsersString = createErrorForInvalidUserList(userKeysResponse, memberList);
+                  return Future.reject(
+                      new SDKError(
+                          new Error(`Failed to create group due to unknown users in user list. Missing user IDs: [${missingUsersString}]`),
```

Comment:
To confirm, isn't `missingUsersString` an array at this point? If so, we should rename it and then call `missingUsers.join(", ")` to convert it to a string instead of relying on whatever JavaScript decides to do to `toString` the array here.